Investigate why updating bindgen to a version newer than 0.60 doesn't emit some bindings

After trying to update `bindings` in `quickjs-wasm-rs` to `0.61` or newer, I get:

```
error[E0432]: unresolved import `quickjs_wasm_sys::size_t`
 --> crates/quickjs-wasm-rs/src/js_binding/value.rs:6:5
  |
6 |     size_t as JS_size_t, JSContext, JSValue, JS_BigIntSigned, JS_BigIntToInt64, JS_BigIntToUint64,
  |     ------^^^^^^^^^^^^^
  |     |
  |     no `size_t` in the root
  |     help: a similar name exists in the module: `time_t`
```

We should probably figure out what's going on so we can continue updating.

--- a/crates/quickjs-wasm-sys/Cargo.toml
+++ b/crates/quickjs-wasm-sys/Cargo.toml
@@ -13,5 +13,5 @@ categories = ["external-ffi-bindings"]
 
 [build-dependencies]
 cc = "1.0"
-bindgen = "0.60"
+bindgen = "0.65.1"
 walkdir = "2"

--- a/crates/quickjs-wasm-sys/Cargo.toml
+++ b/crates/quickjs-wasm-sys/Cargo.toml
@@ -13,5 +13,5 @@ categories = ["external-ffi-bindings"]
 
 [build-dependencies]
 cc = "1.0"
-bindgen = "0.65.1"
+bindgen = "0.64.0"
 walkdir = "2"

--- a/crates/quickjs-wasm-sys/build.rs
+++ b/crates/quickjs-wasm-sys/build.rs
@@ -58,6 +58,7 @@ fn main() {
         .header("wrapper.h")
         .parse_callbacks(Box::new(bindgen::CargoCallbacks))
         .clang_args(&["-fvisibility=default", "--target=wasm32-wasi", &sysroot])
+        .size_t_is_usize(false)
         .generate()
         .unwrap();
 

--- a/install-wasi-sdk.sh
+++ b/install-wasi-sdk.sh
@@ -22,7 +22,7 @@ set -u
 PATH_TO_SDK="crates/quickjs-wasm-sys/wasi-sdk"
 if [[ ! -d $PATH_TO_SDK ]]; then
     TMPGZ=$(mktemp)
-    VERSION_MAJOR="19"
+    VERSION_MAJOR="20"
     VERSION_MINOR="0"
     if [[ "$(uname -s)" == "Darwin" ]]; then
         curl --fail --location --silent https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-${VERSION_MAJOR}/wasi-sdk-${VERSION_MAJOR}.${VERSION_MINOR}-macos.tar.gz --output $TMPGZ

--- a/supply-chain/audits.toml
+++ b/supply-chain/audits.toml
@@ -11,6 +11,11 @@ who = "Jeff Charles <jeff.charles@shopify.com>"
 criteria = "safe-to-deploy"
 version = "1.0.0"
 
+[[audits.proc-macro2]]
+who = "Surma <surma@surma.dev>"
+criteria = "safe-to-deploy"
+version = "1.0.59"
+
 [[audits.quickjs-wasm-rs]]
 who = "Jeff Charles <jeff.charles@shopify.com>"
 criteria = "safe-to-deploy"
@@ -20,3 +25,8 @@ version = "1.0.0"
 who = "Jeff Charles <jeff.charles@shopify.com>"
 criteria = "safe-to-deploy"
 version = "1.0.0"
+
+[[audits.quote]]
+who = "Surma <surma@surma.dev>"
+criteria = "safe-to-deploy"
+version = "1.0.28"

--- a/supply-chain/config.toml
+++ b/supply-chain/config.toml
@@ -77,10 +77,6 @@ criteria = "safe-to-deploy"
 version = "1.3.3"
 criteria = "safe-to-deploy"
 
-[[exemptions.bindgen]]
-version = "0.60.1"
-criteria = "safe-to-deploy"
-
 [[exemptions.bitflags]]
 version = "1.3.2"
 criteria = "safe-to-deploy"
@@ -139,14 +135,6 @@ criteria = "safe-to-deploy"
 
 [[exemptions.clap]]
 version = "2.34.0"
-criteria = "safe-to-deploy"
-
-[[exemptions.clap]]
-version = "3.2.23"
-criteria = "safe-to-deploy"
-
-[[exemptions.clap_lex]]
-version = "0.2.4"
 criteria = "safe-to-deploy"
 
 [[exemptions.cmake]]
@@ -421,10 +409,6 @@ criteria = "safe-to-deploy"
 version = "11.1.3"
 criteria = "safe-to-run"
 
-[[exemptions.os_str_bytes]]
-version = "6.4.1"
-criteria = "safe-to-deploy"
-
 [[exemptions.paste]]
 version = "1.0.6"
 criteria = "safe-to-deploy"
@@ -460,10 +444,6 @@ criteria = "safe-to-deploy"
 [[exemptions.quickcheck]]
 version = "1.0.3"
 criteria = "safe-to-run"
-
-[[exemptions.quote]]
-version = "1.0.15"
-criteria = "safe-to-deploy"
 
 [[exemptions.rand]]
 version = "0.8.5"
@@ -581,10 +561,6 @@ criteria = "safe-to-deploy"
 version = "0.8.0"
 criteria = "safe-to-deploy"
 
-[[exemptions.strsim]]
-version = "0.10.0"
-criteria = "safe-to-deploy"
-
 [[exemptions.structopt]]
 version = "0.3.26"
 criteria = "safe-to-deploy"
@@ -607,10 +583,6 @@ criteria = "safe-to-deploy"
 
 [[exemptions.textwrap]]
 version = "0.11.0"
-criteria = "safe-to-deploy"
-
-[[exemptions.textwrap]]
-version = "0.16.0"
 criteria = "safe-to-deploy"
 
 [[exemptions.thiserror]]

--- a/supply-chain/imports.lock
+++ b/supply-chain/imports.lock
@@ -704,6 +704,12 @@ who = "Johan Andersson <opensource@embark-studios.com>"
 criteria = "safe-to-deploy"
 version = "1.0.58"
 
+[[audits.embark-studios.audits.epaint]]
+who = "Johan Andersson <opensource@embark-studios.com>"
+criteria = "safe-to-deploy"
+violation = "<0.20.0"
+notes = "Specified crate license does not include licenses of embedded fonts if using default features or the `default_fonts` feature. Tracked in: https://github.com/emilk/egui/issues/2321"
+
 [[audits.google.audits.env_logger]]
 who = "George Burgess IV <gbiv@google.com>"
 criteria = "safe-to-run"
@@ -817,6 +823,25 @@ version = "1.1.0"
 notes = "All code written or reviewed by Josh Stone."
 aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
 
+[[audits.mozilla.audits.bindgen]]
+who = "Emilio Cobos Álvarez <emilio@crisal.io>"
+criteria = "safe-to-deploy"
+version = "0.59.2"
+notes = "I'm the primary author and maintainer of the crate."
+aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
+
+[[audits.mozilla.audits.bindgen]]
+who = "Emilio Cobos Álvarez <emilio@crisal.io>"
+criteria = "safe-to-deploy"
+delta = "0.59.2 -> 0.63.0"
+aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
+
+[[audits.mozilla.audits.bindgen]]
+who = "Mike Hommey <mh+mozilla@glandium.org>"
+criteria = "safe-to-deploy"
+delta = "0.63.0 -> 0.64.0"
+aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
+
 [[audits.mozilla.audits.crypto-common]]
 who = "Mike Hommey <mh+mozilla@glandium.org>"
 criteria = "safe-to-deploy"
@@ -884,54 +909,6 @@ aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-ch
 who = "Mike Hommey <mh+mozilla@glandium.org>"
 criteria = "safe-to-deploy"
 delta = "0.3.25 -> 0.3.26"
-aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
-
-[[audits.mozilla.audits.proc-macro2]]
-who = "Nika Layzell <nika@thelayzells.com>"
-criteria = "safe-to-deploy"
-version = "1.0.39"
-notes = """
-`proc-macro2` acts as either a thin(-ish) wrapper around the std-provided
-`proc_macro` crate, or as a fallback implementation of the crate, depending on
-where it is used.
-
-If using this crate on older versions of rustc (1.56 and earlier), it will
-temporarily replace the panic handler while initializing in order to detect if
-it is running within a `proc_macro`, which could lead to surprising behaviour.
-This should not be an issue for more recent compiler versions, which support
-`proc_macro::is_available()`.
-
-The `proc-macro2` crate's fallback behaviour is not identical to the complex
-behaviour of the rustc compiler (e.g. it does not perform unicode normalization
-for identifiers), however it behaves well enough for its intended use-case
-(tests and scripts processing rust code).
-
-`proc-macro2` does not use unsafe code, however exposes one `unsafe` API to
-allow bypassing checks in the fallback implementation when constructing
-`Literal` using `from_str_unchecked`. This was intended to only be used by the
-`quote!` macro, however it has been removed
-(https://github.com/dtolnay/quote/commit/f621fe64a8a501cae8e95ebd6848e637bbc79078),
-and is likely completely unused. Even when used, this API shouldn't be able to
-cause unsoundness.
-"""
-aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
-
-[[audits.mozilla.audits.proc-macro2]]
-who = "Mike Hommey <mh+mozilla@glandium.org>"
-criteria = "safe-to-deploy"
-delta = "1.0.39 -> 1.0.43"
-aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
-
-[[audits.mozilla.audits.proc-macro2]]
-who = "Mike Hommey <mh+mozilla@glandium.org>"
-criteria = "safe-to-deploy"
-delta = "1.0.43 -> 1.0.49"
-aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
-
-[[audits.mozilla.audits.proc-macro2]]
-who = "Mike Hommey <mh+mozilla@glandium.org>"
-criteria = "safe-to-deploy"
-delta = "1.0.49 -> 1.0.51"
 aggregated-from = "https://hg.mozilla.org/mozilla-central/raw-file/tip/supply-chain/audits.toml"
 
 [[audits.mozilla.audits.rustc-hash]]

